sbp_register.py

```diff
@@ -65,9 +91,24 @@ def on_change(self, register):
             return
 
         # Get the region
-        sbp_point_register.store(register, (self.view, self.view.window(), sel[0]))
+        sbp_point_registers.store(register, (self.view, self.view.window(), sel[0], self.view.substr(self.jove.get_encompassing_region())[0:256]))
```

Comment:
Okay I think I know how to do this as I saw a line in the api to get the current line but I also am busy until Sunday. Might have a look then.